index.html

```diff
@@ -32,7 +32,139 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
   <div class="arrow-flex display-flex">
     <span class="arrow">&or;</span>
 </div>
-</section>      
+</section>
+
+<!-- Recent Work Session -->
+<section class="container-recent">
+  <h2 class="recent-work">My Recent Works</h2>
+<div class="project-1 grid-project display-flex">
+  <div class="gray-container"></div>  
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
+</div>
+</div>
```

Comment:
- [ ] Kindly ensure you follow the design guidelines. As you can see in the Figma design the tech stacks are horizontally centered, whereas yours is somewhat aligned to the right. The tech stacks fonts are also bold. This change should be reflected in all the works

- [ ] Kindly add the spacing below 'See project' button as we have in the design. This change should be reflected in all the works

  Your Implementation | Figma Design
  :---:|:---:
  ![ezea-one-work-html](https://user-images.githubusercontent.com/36542986/164949122-ae5f0b70-c138-4309-8396-6c17591aca18.jpg) | ![ezea-one-work-figma](https://user-images.githubusercontent.com/36542986/164949117-749263f9-812d-41cd-be28-2dea00931949.jpg)




- [ ] **[OPTIONAL]** For better code readability, I would recommend you properly indent your code thoughout the page.

Reply:
good, thamks